# size2D msg for Lux

uint16  x
uint16  y
